jenkins/jjb/templates/scenario-tests.yaml

```diff
@@ -146,4 +146,172 @@
+          branches:
+            - '${{TEST_BRANCH}}'
+          basedir: '{repo}'
+          name: ''
+          refspec: '+refs/heads/*:refs/remotes/origin/* +refs/pull/*/head:refs/remotes/origin/pull/*'
+    wrappers:
+      - timeout:
+          name: absolute_timeout
+          timeout: '{build_timeout}'
+          abort: true
+          type: absolute
+      - workspace-cleanup
+    publishers:
+      - robot:
+          output-path: '{repo}/report'
+          pass-threshold: '100'
+          unstable-threshold: '100'
+      - groovy-postbuild:
+          script: |
+              def vc_build = manager.envVars['VC_BUILD']
+              def esx_build = manager.envVars['ESX_BUILD']
+              def vsphere_version = manager.envVars.get('VSPHERE_VERSION')
+              def summary = manager.createSummary("gear2.gif")
+              summary.appendText("Infrastructure Builds:<ul>", false)
+              summary.appendText("<li><b>vSphere Version</b> - ${{vsphere_version}}</li>", false)
+              summary.appendText("<li><b>vCenter</b> - ${{vc_build}}</li>", false)
+              summary.appendText("<li><b>ESXi</b> - ${{esx_build}}</li>", false)
+      - email-ext:
+          success: true
+          aborted: true
+          failure: true
+          content-type: html
+          send-to:
+            - requester
+          subject: 'vmware/{repo} PR Testing - ${{TEST_BRANCH}}'
+          body: |
+              Download link for the test logs is in the <a href="${{BUILD_URL}}/console">build console output</a>
+              <p>
+              ${{VSPHERE_VERSION}} Jenkins Nimbus Scenario Job: <a href="${{BUILD_URL}}">${{BUILD_URL}}</a>
+              <p>
+              Run Report: ${{ROBOT_REPORTLINK}}
+              <p>
+              ${{FILE,path="{repo}/console.log"}}
+
+- job-template:
+    name: 'vic-pr-test'
+    defaults: 'pr-tests'
+    builders:
+      - shell: |
+          #!/bin/bash
+          dest="vic-ci-logs/vmware/${{TEST_BRANCH}}/"
+
+          cat > build.envfile <<ENVS
+          PARALLEL_JOBS=${{PARALLEL_JOBS}}
+          NIMBUS_RETRY_ATTEMPTS=5
+          NIMBUS_RETRY_DELAY=1m
+          LOG_UPLOAD_DEST=${{dest}}
+          GIT_COMMIT=${{GIT_COMMIT}}
+          BUILD_TIMESTAMP=${{BUILD_TIMESTAMP}}
+          BUILD_ID=${{BUILD_ID}}
+          ESX_BUILD=${{ESX_BUILD}}
+          VC_BUILD=${{VC_BUILD}}
+          BUILD_TAG=vic_${{BUILD_NUM}}
+          VCH_BUILD=${{BUILD_NUM}}
+          VCH_BRANCH=${{BUILD_BRANCH}}
+          ENVS
+
+          docker run --rm -v "${{WORKSPACE}}":/go --env-file vic-internal/{repo}-scenario-master-secrets.list --env-file build.envfile -e NIMBUS_USER="${{NIMBUS_USER}}" -e NIMBUS_PASSWORD="${{NIMBUS_PASSWORD}}" --name=${{BUILD_TAG}} gcr.io/eminent-nation-87317/vic-integration-test:1.48 vic-tools/jenkins/jobs/vic-scenario/build.sh ${{VSPHERE_VERSION}} ${{TEST_CASES}}
+
+- job-template:
+    name: 'vic-product-pr-test'
+    defaults: 'pr-tests'
+    builders:
+      - shell: |
```

Comment:
Is there any way to deduplicate this with other jobs?

Reply:
I don't know how to deduplicate this with other jobs.